Add instructions for developing web UI (#32491)
keycloak/keycloak: js/apps/keycloak-server/README.md

Reviewer: Let's split this work off into a separate PR, it is unrelated to the other changes here.

Author: I have put the typo fix in another PR: https://github.com/keycloak/keycloak/pull/32636.

pnpm start --local
```

> If you want to run with `IDELauncher`, make sure you set the following environment variables.

Reviewer: This is not specific to `IDELauncher`, but pertains to all instances where a Keycloak server is started.
```suggestion
If you want to run Keycloak standalone (without the script) against the Vite development server, you can set the following environment variables to achieve the same result:
```

Author: make sense to me

```sh

> export KC_ACCOUNT_VITE_URL=http://localhost:5173
> export KC_ADMIN_VITE_URL=http://localhost:5174
> export KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci

Reviewer: Let's not add the `export` statement here, most people will configure these in their IDE and this makes for easier copy-pasting.
```suggestion
KC_ACCOUNT_VITE_URL=http://localhost:5173
KC_ADMIN_VITE_URL=http://localhost:5174
KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci
```

```

> This environment variables will make your Keycloak admin/account UI retrieve web resource from the local Vite server.

Reviewer: There should be no need to explain this again.

```suggestion
```
